setup.py

```diff
@@ -53,7 +53,6 @@
     'snowballstemmer>=1.1',
     'babel>=1.3,!=2.0',
     'alabaster>=0.7,<0.8',
-    'sphinx_rtd_theme>=0.1,<2.0',
```

Comment:
No, pip doesn't have one. All we can do is to:

* deprecate it in the docs
* throw deprecation warning if user's `html_theme` is equal to `'sphinx_rtd_theme'`

But as far as I get from another pull request, there's a plan to distribute `sphinx_rtd_theme` within `Sphinx` (not as external egg). @birkenfeld could you please comment on this?